Core/include/Acts/TrackFitting/detail/GsfActor.hpp

```diff
@@ -40,7 +40,11 @@ struct GsfResult {
 
   /// The last multi-component measurement state. Used to initialize the
   /// backward pass.
-  std::optional<MultiComponentBoundTrackParameters> lastMeasurementState;
```

Comment:
do we still need the typedef for `MultiComponentBoundTrackParameters`?

Reply:
Hmm this is not a typedef, or am I missing something?